### DISCUSSION TITLE
Proper support for the Kanji from book 3

### DISCUSSION THREAD

**Discussion by m-jepson** [fabd/kanji-koohii#348] [1. Feedback & Feature Requests]
Since I completed the first 2k kanji and moved on to book 3 of ‘Remembering the Kanji,’ I cannot jump to the next Kanji when I click on study, it just goes to Kanji #1. Also, there is no progression indicator, nor a clear list of all the Kanji I’ve learned and have yet to learn. The extra from 3008 are there, but not all the kanji in between.
It would be great if I could go through all 3030 kanji as I’ve done with the first 2k.

**Comment by fabd** [comment]
Hi Michael

yes there is no dashboard-related / progress tracking for RTK Volume 3.

3008-3030 is if you are using old edition. There is less handholding for Volume 3 you'd want to go to the Study page and enter eg. 2043, then use the next arrow to move forward. The "Last Viewed" component in the sidebar can help ease navigation a little bit while studying RTK 3.

You can go through the Study pages, add cards one by one or on the Manage screens (eg `2043-2053`). Teh Restudy list should also work. However once RTK1 is complete there is no progress tracking.

Part of this is due to the fact there is always things to fix/improve and only a small fraction of userbase proceeds to RTK3 since completing RTK1 is a achievement in of itself. And partly it is because it requires a substantial refactor.

One design isue is for example someone might already have studied RTK1 outside of the site and want to work on just RTK3. So there is no logic atm that would allow me to display progress from 0 to 100% of only RTK3. And then this would require allowing the user to switch between RTK1 and RTK3 which is not straightforward, requires implementing multiple tracks/indexes you can be on.

What may be coming sooner I hope is this Study navigation dialog I posted about, it will have a RTK Volume 3 grid so that will help navigate RTK3.

**Reply by m-jepson** [reply]
Thank you for your reply. I understand the difficulties, although it would be great if we had progress tracking for RTK3. I'm coping now by just keeping track of how far I've come separately and jumping to the next kanji to study by simply entering the number. It works, so I can continue my journey.

**Reply by Erfaniom** [reply]
any update on this? im having the same problem, the way it is now is very unintuitive, just adding the percentage/jump to button would suffice, no need for tracking lessons imo

**Reply by fabd** [reply]
Ok I had a quick look, it doesn't seem too complicated. Not sure why I didn't do it before.

**edit:** recoevered my Ubuntu partition, will try to work on this issue next.

Thanks for reminding me about this. Any related notes please post in this new issue:

https://github.com/fabd/kanji-koohii/issues/400
